pallets/nfts/src/tests.rs

```diff
@@ -4112,169 +4129,116 @@ fn clear_collection_metadata_works() {
 }
 
 #[test]
-fn collection_item_works() {
```

Comment:
Moved to the bottom.

Reply:
This creates a lot of noise in the PR. Can we please redo this